### PR TITLE
INITIALIZED GDT

### PR DESCRIPTION
**Pull request by lk1422**
# Pull Request

## Description

Please include a summary of the changes and the related issue (if applicable). Please also include relevant motivation and context. List any dependencies that are required for this change (if applicable).

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

Try to checkoff as much as possible if not everything!

### CODE REVIEW

**Review by joyal-mathew — APPROVED** [review]
Nice style.

File: src/init.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+void set_entry(GdtEntry *entry, u32 base, u32 limit, u8 type, u8 flags){
+    entry->limit      = GET_LOWER_WORD(limit);
+    entry->base_low   = GET_LOWER_WORD(base);
+    entry->base_mid   = GET_MID_BASE(base);
+    entry->type       = type;
+    entry->flags      = (flags << 4) | GET_UPPER_LIMIT(limit);
+    entry->base_upper = GET_UPPER_BASE(base);
+}
+
+void init_gdt(){
```
Comment (Tripplenut) [inline src/init.c:17]: and one here

**Review by Tripplenut — APPROVED** [review]
lgtm

File: src/init.c [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    //Set null descriptor & zero out memory
+    for (int i = 0; i < 3; i++){
```
Comment (Tripplenut) [inline src/init.c:19]: and one here also 💀

**Pull request merged**